frontend/shared/src/components/forms/section/FormSection.tsx

```diff
@@ -14,14 +14,14 @@ const FormSection: React.FC<FormSectionProps> = ({
   header,
   action,
   withoutDivider = false,
-  paddingBottom = true,
+  paddingBottom = false,
   role,
   loading,
   ...rest
 }) => (
   <$Section paddingBottom={paddingBottom}>
     {action && <$Action>{action}</$Action>}
-    {header && <Heading header={header} loading={loading} {...rest} />}
+    {header && <Heading size="s" header={header} {...rest} />}
```

Comment:
thsi probably affects many places in kesäseteli too, could the size be parameterized?